SRC/NBASIC/nbas111e.txt

```diff
@@ -827,11 +822,11 @@ ensemble use of NestorBASIC and InterNestor Lite.
 9.5. ERRORS
 
 Function 86 (InterNestor Lite routine execution)returns error -1 if 
-InterNestor Lite is not installed.
+InterNestor Lite is not resident-loaded.

```

Comment:
Again, no "resident" please.

Reply:
What instead? "Installation" and "installed" sounds really weird to me in this context. We oughta think out something.

"if InterNestor Lite isn't running", perhaps?
"has not been run"?
"has not been loaded" - I write it most of the time, but it's not quite clear for the reader, I guess, so, at the beginning, it should be written as "resident-loaded" a few times.

Reply:
I understand what you mean, strictly speaking "installing" a program is not the same as loading it in memory. But if you look at the documentation of InterNestor (and the program output itself) you'll see that I just use the words "install" and "uninstall", so we should keep the same words here for consistency (and I don't think we need to be that strict, the target audience of this text will understand what we mean) 🙂

Reply:
"Installing into RAM" sounds truly bizarre these days :)
Although I usually advocate authenticity, this one can be really confusing.
So, should I revert ALL the changes concerning "resident"?

Reply:
Yes please. But if you want, you can add a note in section 2, specifying that the term "install" used through the manual means that NestorBASIC has been loaded in memory and is ready to use.

Reply:
Well, it's done.